src/imagery/i.sentinel/i.sentinel.coverage/i.sentinel.coverage.py

```diff
@@ -310,30 +308,31 @@ def main():
             grass.run_command(
                 "v.db.update",
                 map=temp_overlay,
-                column="a_identifier",
-                query_column='a_identifier || "+" ' + "|| b_identifier",
-                where="a_identifier NOT NULL AND " + "b_identifier NOT NULL",
+                column="a_title",
+                query_column='a_title || "+" ' + "|| b_title",
+                where="a_title NOT NULL AND " + "b_title NOT NULL",
                 quiet=True,
             )
             grass.run_command(
                 "v.db.update",
                 map=temp_overlay,
-                column="a_identifier",
-                query_column="b_identifier",
-                where="a_identifier IS NULL",
+                column="a_title",
+                query_column="b_title",
+                where="a_title IS NULL",
                 quiet=True,
             )
             grass.run_command(
                 "v.db.renamecolumn",
                 map=temp_overlay,
-                column="a_identifier,identifier",
+                column="a_title,title",
                 quiet=True,
             )
             columns_dict = grass.parse_command("v.info", map=temp_overlay, flags="c")
             drop_columns = [
                 col.split("|")[1]
                 for col in columns_dict
-                if col.split("|")[1] not in ["cat", "identifier"]
+                if col.split("|")[1]
+                not in ["cat", "title"]  # What does cat refer to here?
```

Comment:
cat is the geometry identifier in GRASS vector maps, so, like the id in other formats